src/main/resources/docgen/templates/msmarco-passage-bm25-b8.template

```diff
@@ -0,0 +1,52 @@
+# Anserini Regressions: MS MARCO Passage Ranking With Quantized BM25 Weights
+
```

Comment:
Add something like "Models: BM25 with quantized weights (8 bits)" to be consistent with:
https://github.com/castorini/anserini/blob/master/docs/regressions-msmarco-passage.md